pkgs/games/factorio/default.nix

```diff
@@ -9,6 +9,8 @@
 assert releaseType == "alpha"
     || releaseType == "headless"
     || releaseType == "demo";
+assert stdenv.system == "x86_64-linux"
```

Comment:
Why isn't this just using `meta.platforms`? The derivation does appear to set `meta.platforms` to only support `x86_64-linux` too.

Reply:
I'm also not sure how an assertion that `stdenv.system` is linux would work on Darwin...

Reply:
Oh, I see, sorry, I missed your intro paragraph. Nevermind!